src/external/druidExternal.ts

```diff
@@ -1329,7 +1329,16 @@ return (start < 0 ?'-':'') + parts.join('.');
       return postAgg;
     }
 
-    public makeStandardAggregation(name: string, filterAction: FilterAction, aggregateAction: Action): Druid.Aggregation {
+    public makeNativeAggregateFilter(name: string, filterExpression: Expression, aggregator: Druid.Aggregation): Druid.Aggregation {
```

Comment:
all the other Druid Aggregation makers seem to be named make***Aggregation but this one doesn't follow that, is this because it's special?